lightning/src/routing/gossip_checking.rs

```diff
@@ -477,4 +477,28 @@ impl PendingChecks {
 			}
 		}
 	}
+
+	const MAX_PENDING_LOOKUPS: usize = 32;
```

Comment:
Should this be user-adjustable, depending on their needs and available resources?